### PR TITLE
Finagle's 7th Birthday

### PR DESCRIPTION
**Pull request by kevinoliver**

### CODE REVIEW

File: source/blog/2017-10-18-happy-7th-birthday.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+The pace of development remains high — with over 1,600 commits in the
+past year across [Finagle][finagle], [Finatra][finatra], [Util][util], [Scrooge][scrooge], and [TwitterServer][twitterserver].
+It is unlikely to slow down with 13 devs on the core team. Externally
+the community [adoption][adoption] is broad, spawned a [company][company], and inspired
+versions in [C++][cpp] and [Rust][rust].
+
+With a wide user base numbering into the hundreds of services, our
+development philosophy has changed focus over the years. There has
+been a shift to capabilities that service owners get for free — simply
+redeploy with the latest version and voil! Examples include retry
```
Comment (mosesn) [inline source/blog/2017-10-18-happy-7th-birthday.md]: voila

File: source/blog/2017-10-18-happy-7th-birthday.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+budgets, HTTP/2, improved throughput, and admission control with
+automatic retries among others.
+
+While you'd now consider a robust RPC library table stakes in
+microservice deployments, that was not the case in 2010. Were proud of
```
Comment (mosesn) [inline source/blog/2017-10-18-happy-7th-birthday.md]: we're

File: source/blog/2017-10-18-happy-7th-birthday.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+Finagles consistent progress and excited about Future\[Finagle\].
```
Comment (mosesn) [inline source/blog/2017-10-18-happy-7th-birthday.md]: ffti: `Future[Finagle]`

Comment (mosesn) [reply source/blog/2017-10-18-happy-7th-birthday.md]: Finagle's

File: source/blog/2017-10-18-happy-7th-birthday.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+Heres to the next 7 years,
```
Comment (mosesn) [inline source/blog/2017-10-18-happy-7th-birthday.md]: Here's

**Comment by kevinoliver** [comment]
thanks @mosesn. copy paste or something removed any fancy formatting. fixed em.

**Pull request merged**